### DISCUSSION TITLE
iOverlander

### DISCUSSION THREAD

**Discussion by luigifab** [osmandapp/OsmAnd#20228] [Ideas]
Is there a plugin for displaying iOverlander POIs into OsmAnd?

**Comment by rachellawson** [comment]
Not aware of one but it is a damn fine idea!

**Comment by mariush444** [comment]
There is converter to move exoprted gpx fromm iOverlander to osmand with icons
https://mariush444.github.io/Osmand-tools/
but exported gpx contains only the 1st description of a point (that is rule)

https://github.com/osmandapp/OsmAnd/issues/19971

**Reply by luigifab** [reply]
Nice! but I'm currenrly traveling and I prefer to do something else. Maybe next year I'm going to learn how to do a plugin.
